src/OpenLoco/Graphics/Gfx.cpp

```diff
@@ -1,4 +1,5 @@
 #include "Gfx.h"
+#include "../Company.h"
```

Comment:
For a follow-up after this PR: I think we should move the `ColourScheme` type to `Types.hpp`, so we can get rid of the `Company.h` include.

Reply:
PR created for this: https://github.com/OpenLoco/OpenLoco/pull/753